tests/libcxx2/Makefile

```diff
@@ -0,0 +1,110 @@
+TOP=$(abspath ../..)
+include $(TOP)/defs.mak
+
+ifdef STRACE
+OPTS = --strace
+endif
+
+APPDIR=$(CURDIR)/appdir
+APPBUILDER=$(TOP)/scripts/appbuilder
+
+ROOTFS=$(CURDIR)/rootfs
+ROOTHASH=$(SUBOBJDIR)/roothash
+
+LIBCXX_TESTS_DIR=llvm-project/libcxx/test/
+
+# batching passed testfiles here helps to remove large heap memory requirement from 3gb 
+# to 1gb
+ALLTESTS=tests.all
+export TIMEOUT=10000
+
+all:
+	$(MAKE) clean
```

Comment:
"clean" should be removed. This would cause trouble for "make -j", in which the dependency list of "all" are built in parallel.

Same applied to `tests/libcxx/Makefile`.

Reply:
Thanks for letting me know, removed this from this PR. Will put in another PR for changes on libcxx!

Reply:
Please fix that in this PR.

Reply:
Alrighty, added